registry/scripts/fix_updated_at.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+"""
+Rebuilds the full-text search index (to fix the language).
+"""
+
+import sys
+
+from quilt_server import db
+
+def main(argv):
+    result = db.engine.execute('''
```

Comment:
i guess batching is overkill even for prod?

Reply:
Yeah... It's still under 1000 rows.